statsmodels/survey/survey_model.py

```diff
@@ -0,0 +1,50 @@
+import numpy as np
+
+class SurveyModel(object):
+
+    def __init__(self, design, model_class, init_args={}, fit_args={}):
+        self.design = design
+        self.model = model_class
+        self.init_args = dict(init_args)
+        self.fit_args = dict(fit_args)
+
+    def fit(self, y, X, replicates=None):
+        self.init_args["weights"] = self.design.weights
+        self.params = self._get_params(y, X)
+        if replicates is None:
```

Comment:
Actually, this try/except is just trying to get `design.nclust`, which is not set if the user assigns rep_weights. I will just change init to set `self.nclust = self.rep_weights.shape[1]` when they are provided